Kitodo/src/test/java/org/goobi/io/BackupFileRotationTest.java

```diff
@@ -102,39 +102,41 @@ public void shouldWriteTwoBackupFiles() throws IOException {
 
     @Test
     public void initialContentShouldEndUpInSecondBackupFileAfterTwoBackupRuns() throws IOException {
-        String content1 = "Test One.";
-        int numberOfBackups = 2;
+        String contentOne = "Test One.";
+        final int kNumberOfBackups = 2;
```

Comment:
Rename of this variable is not needed. Add of _k_ letter is actually against the coding guideline as _k_ has no meaning here.

Reply:
k is indicating a constant. This is called “Apps Hungarian” style which basically means prefixing variable names of the same `class` with well known letters to indicate their content. (Must not be confused with “Systems Hungarian” style which is a misuse of the further.) Normally, constants should be written uppercase with underlines, but checkstyle sees them as abbreviations, so no way.

Reply:
As mentioned above: I did not agree with your changes to made names unreadable. Maybe "k" is clear to you that this in indicate a constant value but not to other people. If you want to use constant values then use them properly or stay with the usage scenario.